src/data/ab-attrs/post-dancing-move-ab-attr.ts

```diff
@@ -33,11 +32,11 @@ export class PostDancingMoveAbAttr extends PostMoveUsedAbAttr {
       if (!simulated) {
         if (move.getMove().isSelfStatusMove()) {
           // If the move is a SelfStatusMove (ie. Swords Dance), the Dancer should replicate it on itself
-          globalScene.unshiftPhase(new MovePhase(pokemon, [pokemon.getBattlerIndex()], move, true, true));
+          globalScene.useMove({ pokemon, targets: [pokemon.getBattlerIndex()], move, followUp: true, when: "eager" });
```

Comment:
```suggestion
          globalScene.useMove({ pokemon, targets: [pokemon.getBattlerIndex()], move, followUp: true, ignorePP: true, when: "eager" });
```
If I understand correctly the second true before was for `ignorePP`?

Reply:
True. I'm surprised how much I missed here.

https://github.com/Despair-Games/poketernity/blob/d507828e4c4d72dda03ac6cd4178e5d86c1ceade/src/data/ab-attrs/post-dancing-move-ab-attr.ts#L36

Reply:
It's `ignorePp` though (not `*PP`)

Reply:
All good, it's a lot of changes. Finding this stuff makes me feel better about spending so much time reviewing too 😂